tests/fixtures/points.py

```
@@ -1,53 +1,105 @@
 import random
 import uuid
 from typing import Union
-
+from qdrant_client.local.distances import calculate_distance
 import numpy as np
+from qdrant_client.http import models
 
 from qdrant_client._pydantic_compat import construct
 from qdrant_client.http import models
```

Ruff (0.8.2)

Redefinition of unused `models` from line 6

Remove definition: `models`

(F811)